src/transformers/pipelines/visual_question_answering.py

```diff
@@ -33,17 +73,28 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.check_model_type(MODEL_FOR_VISUAL_QUESTION_ANSWERING_MAPPING)
 
-    def _sanitize_parameters(self, top_k=None, padding=None, truncation=None, **kwargs):
+    def _sanitize_parameters(self, top_k=None, padding=None, truncation=None, words=None, boxes=None, **kwargs):
         preprocess_params, postprocess_params = {}, {}
         if padding is not None:
             preprocess_params["padding"] = padding
         if truncation is not None:
             preprocess_params["truncation"] = truncation
         if top_k is not None:
             postprocess_params["top_k"] = top_k
+        if words is not None or boxes is not None:
+            if words is None or boxes is None:
+                raise ValueError("Must provide both words and boxes if providing either")
+            preprocess_params["words"] = words
+            preprocess_params["boxes"] = boxes
+
         return preprocess_params, {}, postprocess_params
 
-    def __call__(self, image: Union["Image.Image", str], question: str = None, **kwargs):
+    def __call__(
+        self,
+        image: [Optional[Union["Image.Image", str]]] = None,
```

Comment:
This needs to be set at call time. It's a visual question answering so it does require an image.

Reply:
Will adjust per our discussion below. I think what we should do is make `image` a required argument, and then (without making it too obvious), allow an advanced user to pass `None` (or another empty-like value) and handle it gracefully within the implementation.